.github/workflows/publish-docs.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish Docs
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - "docs/**"
+      - ".github/workflows/publish-docs.yml"
+      - "Makefile"
+
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: |
+        make st2
+        sudo apt-get update
+        # python3-dev is already available
+        # sudo apt install python3-dev
+        sudo apt install libldap2-dev 
+        sudo apt install libsasl2-dev
+        sudo pip3 install virtualenv
+        make docs
+
+        # turn off jekyll so that url routing works properly with underscores
+        touch docs/build/html/.nojekyll
+    - name: Deploy to GitHub Pages
+      uses: crazy-max/ghaction-github-pages@v3
```

Comment:
There is another repo that is using `peaceiris/actions-gh-pages@v3` https://github.com/StackStorm/stackstorm.com/blob/3198d86662be419cb5b64197d2223ca225440f20/.github/workflows/gh-pages.yaml#L30-L32

Looking at [`crazy-max/ghaction-github-pages`](https://github.com/crazy-max/ghaction-github-pages) here (300 stars) vs [`peaceiris/actions-gh-pages`](https://github.com/peaceiris/actions-gh-pages) in another repo (3K stars), I think it makes sense to go with a more popular one that we're using already.

Reply:
I agree. this has been done.